engine/src/test/scala/cromwell/engine/backend/jes/RefreshTokenModeSpec.scala

```diff
@@ -10,7 +10,7 @@ import cromwell.util.EncryptionSpec
 import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 import org.specs2.mock.Mockito
 
-class JesBackendSpec extends FlatSpec with Matchers with Mockito with BeforeAndAfterAll with WorkflowDescriptorBuilder {
+class RefreshTokenModeSpec extends FlatSpec with Matchers with Mockito with BeforeAndAfterAll with WorkflowDescriptorBuilder {
   val testWorkflowManagerSystem = new CromwellTestkitSpec.TestWorkflowManagerSystem()
```

Comment:
1) We no longer need to spin up a whole workflow manager system if we just need an actor system. There's now [`TestKitSuite`](https://github.com/broadinstitute/cromwell/blob/6af5611c27c678bda2b55950e1a3e37c1c766462/core/src/test/scala/cromwell/core/TestKitSuite.scala) for that.

2) This spec doesn't even use the thread pool spun up inside the WMS / actor system. So remove the `testWorkflowManagerSystem` and related cruft, delete all usage of the `BeforeAndAfterAll` trait, and while there I don't think `Mockito` and `workingDisk` are used either.